packages/opentelemetry-plugin-http/package.json

```diff
@@ -39,19 +39,29 @@
     "access": "public"
   },
   "devDependencies": {
+    "@opentelemetry/scope-async-hooks": "^0.0.1",
```

Comment:
this should not be a dependency of the plugin (and it's actually a deps of node-tracer)

Reply:
I use it in tests for making an instance of `NodeTracer`.